omod/src/main/webapp/pages/metadata/encounters/encountertypes/templates/edit.gsp

```diff
@@ -37,3 +37,71 @@
     </p>
 </form>
 
+<hr>
+<fieldset class="right" ng-show="encounterType.uuid">
+    <legend>${ui.message('view.Encounters')}</legend>
+    <p>
+        ${ui.message('general.uuid')}: {{ encounterType.uuid }}
+    </p>
+    <p>
+        ${ui.message('general.createdBy')}:
+        {{ encounterType.auditInfo.creator | omrsDisplay }}
+        ${ui.message('general.onDate')}
+        {{ encounterType.auditInfo.dateCreated | serverDate }}
+    </p>
+    <p ng-show="encounterType.changedBy">
+        ${ui.message('general.changedBy')}:
+        {{ encounterType.auditInfo.changedBy | omrsDisplay }}
+        ${ui.message('general.onDate')}
+        {{ encounterType.auditInfo.dateChanged | serverDate }}
+    </p>
+</fieldset
+
+
+
+<hr>
+<fieldset class="right" ng-show="encounterType.uuid">
```

Comment:
Why are you adding these sections  for audit info?

Reply:
@wluyima  I was thinking of separating concerns on the same page i.e put the different functionalities in different fieldsets

Reply:
However am re-editing the page to remove that,i was thinking of introducing another page,but looks like that will be reinventing the wheel as most of the Encounter attributes have been taken care of @edit.gsp